src/components/PolicyTopologyPage.tsx

```diff
@@ -358,12 +358,60 @@ const customLayoutFactory = (type: string, graph: any): any => {
   });
 };
 
+interface PolicyConfig {
+  key: string;
+  displayName: string;
+}
+const RESOURCE_POLICIES_MAP: Record<string, PolicyConfig[]> = {
```

Comment:
I recommend following the same naming convention as the rest of the consts. Caps are usually used for truly constant values. This const is more of a lookup